dal/src/test/kotlin/com/home/generator/JooqGenerator.kt

```diff
@@ -0,0 +1,55 @@
+package com.home.generator
+
+import org.jooq.util.GenerationTool
+import org.jooq.util.jaxb.*
+import org.jooq.util.jaxb.Target
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.io.FileInputStream
+import java.util.*
+
+class JooqGenerator {
+
+    @Test
+    @Disabled
```

Comment:
I'd argue that turning codegen on/off using a command-line variable would be easier to understand. Like how it's described in [Gradle guide](https://guides.gradle.org/migrating-from-maven/#maven_profiles_and_properties)

Would be interesting to hear other opinions though

Reply:
let's discuss that and keep as task to improve in backlog